kubernetes/ibm-ppc64le/prow/secretstore-ibm-k8s.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ClusterSecretStore
+metadata:
+  name: secretstore-ibm-k8s
+spec:
+  provider:
+    ibm:
+      serviceUrl: "https://3297fd32-6322-45e2-af3f-00b1a5af3565.us-south.secrets-manager.appdomain.cloud"
+      auth:
+        secretRef:
+          secretApiKeySecretRef:
+            name: ibm-sm-apikey
+            key: API_KEY
+            namespace: test-pods
```

Comment:
is this a requirement to have this secret in the test-pods? if not lets avoid this.